Spigot-Server-Patches/0611-Add-warning-for-servers-not-running-on-Java-11.patch

```diff
@@ -60,14 +69,22 @@ index 0000000000000000000000000000000000000000..afe07487b1e7337293f7147619a6f10f
 +    }
 +}
 diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
-index 0108a1a68572df562349688e93f8134cb14d6116..12d781f4ae6a46d0885972035870d5b5b62f5b8b 100644
+index 0108a1a68572df562349688e93f8134cb14d6116..0d3d8fde1d3ae94ec79b36ad40997c39100d252a 100644
 --- a/src/main/java/net/minecraft/server/MinecraftServer.java
 +++ b/src/main/java/net/minecraft/server/MinecraftServer.java
-@@ -950,6 +950,8 @@ public abstract class MinecraftServer extends IAsyncTaskHandlerReentrant<TickTas
+@@ -69,6 +69,7 @@ import org.bukkit.craftbukkit.Main;
+ import org.bukkit.event.server.ServerLoadEvent;
+ // CraftBukkit end
+ import co.aikar.timings.MinecraftTimings; // Paper
++import io.papermc.paper.util.PaperJvmChecker; // Paper
```

Comment:
Unnecessary import, the class is only used 1 time.

Reply:
actually, it is necessary, you get compile errors if you don't do imports on any of the classes in the paper io package, because there is an `io` nms class that throws everything off. its really annoying

Reply:
Jesus fucking christ. You're right, I forgot about that nonsense. Carry on...

Reply:
rip tokyo